frontend/components/home/js/home.component.ts

```diff
@@ -254,7 +254,7 @@ class HomeController implements ng.IController {
 						});
 					}
 
-				} else if (currentData.initialiser && !currentData.username) {
+				} else if (!currentData.username) {
```

Comment:
Are you completely sure you can remove currentData.initialiser? I remember this being very fragile!

Reply:
Line 243 is `if (!currentData.initialiser) {`, so I'd hope `else` means the opposite of that.

Reply:
Oh dear